components/block-description/index.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I actually tested it as a functional component in that folder at first but wasn't sure if it's correct / the best approach. Happy to move it.